cypress/support/pages/userPage.js

```diff
@@ -15,7 +15,11 @@ export function fillUpSetPassword(password) {
     .waitForRequestAndCheckIfNoErrors("@setPassword");
 }
 
-export function fillUpUserDetails(firstName, lastName, email) {
+export function fillUpUserDetailsAndAddFirstPermission(
```

Comment:
The part of code in this function is the same as in fillUpOnlyUserDetails(). You can do something like

fillUpUserDetailsAndAddFirstPermission(){
    fillUpOnlyUserDetails()

Reply:
ok